### DISCUSSION TITLE
Zigbee2Mqtt, intervallo di polling dati

### DISCUSSION THREAD

**Discussion by FabioEight** [indomus/forum#707] [ZigBee, Z-Wave, Thread, Matter]
Ciao, 
ho installato un sensore di umidità/temperatura Sonoff ed l'ho integrato in Zigbee2Mqtt. 
Ho notato che spara una quantità fin eccessiva di valori, controllando il grafico di HA.
Controllando su Indomus, ho seguito la guida per creare dei filtri ( https://indomus.it/guide/ridurre-le-scritture-dati-di-sensori-ipertrofici-su-home-assistant/ ) per ottenere grafici più lineari.
Verificando su Zigbee2Mqtt nella sezione Reporting del sensore, ho notato che ci sono le opzioni "Min rep interval", "Max rep interval" e "Min rep change", forse è possibile variare questi parametri per ottenere polling a distanza leggermente maggiore, oppure aumentare l'intervallo di aggiornamento in caso di variazione parametro? Es. inviare le info aggiornate in caso di variazione di 0.1°C di temperatura al posto del 0.01°C, oppure dello 0.1% di umidità piuttosto che 0.01%.
Grazie per il supporto!

**Comment by snakuzzo** [comment]
Quello che puoi fare sul device (incluso il polling interurlval, lo trovi scritto in documentazione.
Nel tuo caso parliamo del Sonoff SNZB-02 immagino, quindi puoi trovare le info [QUI](https://www.zigbee2mqtt.io/devices/SNZB-02.html)

**Reply by FabioEight** [reply]
> Quello che puoi fare sul device (incluso il polling interurlval, lo trovi scritto in documentazione.
> Nel tuo caso parliamo del Sonoff SNZB-02 immagino, quindi puoi trovare le info [QUI](https://www.zigbee2mqtt.io/devices/SNZB-02.html)

Ho provato ad incrementare i valori, ma in seguito all'applica, vengono ripristinati i precedenti

` Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'Bind 0x00124b0023c2a303/1 msTemperatureMeasurement from '0x00212effff061271/1' failed (waiting for response TIMEOUT)'`

**Reply by snakuzzo** [reply]
Ma se leggi la doc che ti ho linkato, non mi pare tu possa modificare quei parametri

**Reply by Scialla** [reply]
a memoria, mi sembra che il polling sia modificabile solo su device alimentati a corrente e non a batteria
